src/main/java/com/happystays/cqrs/core/events/EventModel.java

```diff
@@ -1,6 +1,6 @@
 package com.happystays.cqrs.core.events;
 
```

Comment:
Please keep PEventModel and EventModel inside com.happystays.cqrs.core.events -> eventstoreModel -> EventModel.Java